src/box/lua/tuple.h

```diff
@@ -33,6 +33,12 @@
 #include <stddef.h>
 #include "trivia/util.h"
 
+#include "lua/utils.h"
+#include "lua/trigger.h"
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
```

Comment:
Nits:

1. I guess `lua.h` is enough.
2. System headers should go first.

Reply:
I can't get it to work if I remove some of the includes or change their order. Linking (!) fails with a lot of errors like
```lua
undefined reference to `lua_settop(lua_State*, int)'
```